modules/sw360/src/main/java/org/eclipse/sw360/antenna/sw360/utils/RestUtils.java

```diff
@@ -123,6 +124,9 @@
             externalIds.put(SW360Attributes.RELEASE_EXTERNAL_ID_COPYRIGHTS, sw360Release.getCopyrights());
         }
 
+        if(externalIds.isEmpty()) {
+            return Collections.emptyMap();
```

Comment:
this only returns an immutable map instead off the empty `externalIds` map? Why is this here?

Reply:
Will return to null, as it was before- Since this is how it is handled when called.

Reply:
but if `externalIds` would be null, it would throw a NPE here?